lib/pluginmanager/update.rb

```diff
@@ -6,6 +6,7 @@ class LogStash::PluginManager::Update < LogStash::PluginManager::Command
   REJECTED_OPTIONS = [:path, :git, :github]
 
   parameter "[PLUGIN] ...", "Plugin name(s) to upgrade to latest version", :attribute_name => :plugins_arg
+  option "--[no-]verify", :flag, "verify plugin validity before installation", :default => true
```

Comment:
Sorry, I am still confused. is the `--no-verify` option required for offline installation?